starknet_py/net/schemas/common.py

```diff
@@ -336,3 +337,34 @@ class StorageEntrySchema(Schema):
     def make_dataclass(self, data, **kwargs):
         # pylint: disable=no-self-use
         return StorageEntry(**data)
+
+
+class Revision(Enum):
+    """
+    Enum representing the revision of the specification to be used.
+    """
+
+    V0 = 0
+    V1 = 1
+
+
+class RevisionField(fields.Field):
+    def _serialize(self, value: Any, attr: str, obj: Any, **kwargs):
+        if value is None:
+            return str(Revision.V0.value)
+        if value == Revision.V0:
+            return str(value.value)
+        return int(value.value)
```

Comment:
nit: Casting is not required here
```suggestion
        return value.value
```